sdk/tdf.go

```diff
@@ -863,6 +863,10 @@ func (r *Reader) doPayloadKeyUnwrap(ctx context.Context) error { //nolint:gocogn
 
 			unencryptedMetadata = metaData
 		}
+
+		if r.manifest.EncryptionInformation.KeyAccessObjs[0].SplitID == "" {
```

Comment:
If this is for treating `sid`-less multi-kao files as 'or', you should instead remove the `mixedSplits` processing.

Reply:
@dmihalcik-virtru is that what you meant (**remove the mixedSplits**)?

Reply:
I think you can also get rid of the `if mixedSplits && ...` block below and remove the variable entirely

Reply:
If removing it is causing a regression, you can add this back and I'll take a look after this gets merged in